.betterer.results

```diff
@@ -4373,6 +4373,9 @@ exports[`better eslint`] = {
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
       [0, 0, 0, "Unexpected any. Specify a different type.", "1"]
     ],
+    "public/app/features/explore/TableContainer.tsx:5381": [
+      [0, 0, 0, "Do not use any type assertions.", "0"]
+    ],
```

Comment:
👍🏻 

Not great to see new items added here, but this is better than the existing `// @ts-ignore`. Thanks!